sim/fold_coverage_test.py

```diff
@@ -0,0 +1,30 @@
+# Load a 
```

Comment:
Are we keeping this file?

Reply:
yes to this one! I think this test might be useful down the line. Especially when we want to confirm performance tolerance on samples. We don't have control over the level of fold-coverage in real life.